### DISCUSSION TITLE
Specific number of metals

### DISCUSSION THREAD

**Discussion by ashkanr65** [The-OpenROAD-Project/OpenLane#1145] [General] [answered]
Hi, 
I was wondering if it is possible to run OpenLane with a specific number of metals? Currently, it uses "li" with 5 metal layers (metal1, metal2, metal3, metal4, metal5) from sky130hd. Is it possible to decrease this number to 2 metal layers? (for example "li" + "metal1" or "metal1" + "metal2")

**Comment by maliberty** [comment]
If you set the max routing layer for global & detailed routing it should limit it.   You would also need to think about your pdn setup.

Just curious why you want to do this.

**Reply by ashkanr65** [reply]
I am working on a new PDK, but I have limitations on layers. So I want to know how many layers I can use for the Openlane.

Sorry to ask, but can you be more specific? Which setup should I change for PDN setup?

**Comment by maliberty** [comment, marked as answer]
RT_MAX_LAYER controls the max routing layer.

PDN is controlled by:
FP_PDN_RAILS_LAYER
FP_PDN_LOWER_LAYER
FP_PDN_UPPER_LAYER
but I'm not sure how you want your grid to look with only two layers.
